lib/puppet/pops/validation.rb

```diff
@@ -91,7 +91,7 @@ def label_provider
   # @api public
   #
   class SeverityProducer
-    @@severity_hash = {:ignore => true, :warning => true, :error => true, :deprecation => true }
+    SEVERITIES = { ignore: true, warning: true, error: true, deprecation: true }
```

Comment:
Can you freeze the hash `{ ... }.freeze`?

Reply:
absolutely